lib/Backend/Lower.cpp

```diff
@@ -8197,8 +8200,12 @@ Lowerer::LowerLdElemI(IR::Instr * instr, IR::JnHelperMethod helperMethod, bool i
     {
```

Comment:
Done.

---
In reply to: [67771530](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r67771530) [](ancestors = 67771530)